.github/workflows/multi-gcc.yml

```diff
@@ -25,133 +25,399 @@ jobs:
     - name: Checkout submodules
       run: git submodule update --init
 
-    - name: GCC 6.2.1 Debug
-      if: always()
+    - name: Host Release
+      run: cd ${{github.workspace}}; mkdir -p build; rm -rf build/*; cd build; cmake ../ -DPICO_SDK_TESTS_ENABLED=1 -DCMAKE_BUILD_TYPE=Release -DPICO_NO_PICOTOOL=1 -DPICO_PLATFORM=host; make --output-sync=target --no-builtin-rules --no-builtin-variables -j$(nproc)
+
+    - name: Host Debug
+      run: cd ${{github.workspace}}; mkdir -p build; rm -rf build/*; cd build; cmake ../ -DPICO_SDK_TESTS_ENABLED=1 -DCMAKE_BUILD_TYPE=Debug -DPICO_NO_PICOTOOL=1 -DPICO_PLATFORM=host; make --output-sync=target --no-builtin-rules --no-builtin-variables -j$(nproc)
+
+    - name: GCC 6.2.1 Debug Pico W
+      if: ${{ !cancelled() }}
+      shell: bash
+      run: cd ${{github.workspace}}; mkdir -p build; rm -rf build/*; cd build; cmake ../ -DPICO_SDK_TESTS_ENABLED=1 -DCMAKE_BUILD_TYPE=Debug -DPICO_NO_PICOTOOL=1 -DPICO_BOARD=pico_w -DPICO_PLATFORM=rp2040 -DPICO_TOOLCHAIN_PATH=/opt/arm/gcc-arm-none-eabi-6_2-2016q4; make --output-sync=target --no-builtin-rules --no-builtin-variables -j$(nproc)
```

Comment:
Doesn't the use of `-DPICO_BOARD=pico_w` automatically imply `-DPICO_PLATFORM=rp2040` ? (although I guess there's no harm in being verbose? :slightly_smiling_face: )

Reply:
yeah this is generated by a script